### DISCUSSION TITLE
ETF는 지원이 안되나요?

### DISCUSSION THREAD

**Discussion by ryukidong** [mrchypark/tqk#44] [Q&A] [answered]
삼성전자 같은 개별주는 잘 되는것 같은데 
069500	KODEX 200
143850	TIGER 미국 S&P5
이런  ETF 는 안된는것 같은데   ETF 는 지원을 안하나요?

**Comment by mrchypark** [comment, marked as answer]
@ryukidong 안녕하세요 살펴보니 제가 주식 종목만 가져오고 있는 것 같습니다. 알려주셔서 감사합니다.

krx 사이트에서 정보를 가져오고 있는데요. 지수, 파생등 정보가 많네요. 이슈 등록하고 작업해보도록 하겠습니다.

현재는 지원하지 않는 것이 맞습니다.

**Reply by ryukidong** [reply]
답변 감사합니다.
제가 R로 자산배분 관련 백테스트 하는데 관심이 많아서 ETF 정보가 필요한데 한국 ETF는 다른 라이브러리들도 잘 지원을 안하네요.
나중에 업데이트 되면 알려주세요.
감사합니다.

**Reply by mrchypark** [reply]
https://github.com/mrchypark/tqk/issues/45

이슈 생성해 두었습니다. 완료 되면 알람이 갈거에요. 

이렇게 의견주신 덕분에 더 패키지를 좋게 만들 수 있어서 저야말로 감사드립니다.
